manage_breast_screening/record_a_mammogram/forms.py

```diff
@@ -0,0 +1,43 @@
+from django import forms
+
+
+class ScreeningAppointmentForm(forms.Form):
+    next_step = forms.ChoiceField(
```

Comment:
The 'next step' concept brings to mind the path or identifier for the following page in the journey, so perhaps something like:

```suggestion
    choice = forms.ChoiceField(
```

Reply:
I've gone with decision just to avoid the clash with `ChoiceField` and `choices` which we might have multiple of.